docs/meshstack.meshmarketplace.metering.md

```diff
@@ -67,6 +67,10 @@ The following sections detail how meshStack interprets OSB API plan [cost object
 
 See [Metrics-based Metering](meshstack.meshmarketplace.metrics-metering.md) for details about how to charge your services usage-based. This could be relevant for you if you want to charge e.g. based on actual storage or memory used by the service. Further use-cases are mentioned in the linked chapter.
 
+> You have to make sure that the metric endpoints are available reliably. When Tenant Usage Reports are
+> finalized (which is usually four days after the end of the month), metrics that haven't been received until that point in time for the previous month won't be
+> considered in the Usage Reports and Chargeback Statements.
```

Comment:
we should mention that the point for finalization is tailored to the org and theyshould get in touch with their cloud foundation team

Reply:
ok. i added this info